Packages/com.unity.inputsystem/InputSystem/Actions/InputActionState.cs

```diff
@@ -2048,6 +2056,32 @@ private void ProcessInteractions(ref TriggerState trigger, int interactionStartI
             }
         }
 
+        private void ProcessHoldInteractions(ref TriggerState trigger, int interactionStartIndex, int interactionCount)
+        {
+            var context = new InputInteractionContext
+            {
+                m_State = this,
+                m_TriggerState = trigger
+            };
+
+            for (var i = 0; i < interactionCount; ++i)
+            {
+                var index = interactionStartIndex + i;
+                var state = interactionStates[index];
+                var interaction = interactions[index];
+                if (interaction is not HoldInteraction)
```

Comment:
This might not be correct, all interaction could be affected depending on the settings.
User's custom interaction might also be affected.

Reply:
Could you elaborate please? I'm not sure I understand how they might be affected